tools/api-generator/string.go

```diff
@@ -82,3 +82,38 @@ func addNewLineForBlockLevelElements(n *html.Node, buf *bytes.Buffer) {
 		}
 	}
 }
+
+// Simplified port of:
+// - https://github.com/Redocly/redoc/blob/59ee73fefa8e8edb398940076bdd721fc284caa3/src/utils/helpers.ts#L123
+// - https://github.com/simov/slugify/blob/master/slugify.js
+//
+// Note: this does not handle special characters, which is fine for now as none of or tags or OperationIDs use special characters.
```

Comment:
I was under the wrong impression that special characters where not allowed for Tags and OperationID. I follow up this internally